From 55eae4ded50cd48acc0cd6fd103655afe6b05458 Mon Sep 17 00:00:00 2001
Message-Id: <55eae4ded50cd48acc0cd6fd103655afe6b05458.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Thu, 24 Jul 2014 19:09:41 +0530
Subject: [PATCH 252/312] Revert "FROM_UPSTREAM [VPG]: drm/i915:
 Disable/Enable PM Intrrupts based on the current
 freq"

This reverts commit 3dcef67a6f7b6b0908586c39ed10b0e86dec7afd.

We already have the upstream implementation merged.

commit 2876ce734417b1893d48e86c9ade9100e1fcc297
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Fri Mar 28 08:03:34 2014 +0000

    drm/i915: Mask PM/RPS interrupt generation based on activity

Issue: GMIN-3334
Change-Id: Iad95b586ad2036cccb636751b96554c9c241d616
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_irq.c |    1 -
 drivers/gpu/drm/i915/intel_pm.c |    3 ---
 2 files changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index 2ed4a57..7fff34e 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -1487,7 +1487,6 @@ static void gen6_pm_rps_work(struct work_struct *work)
 			    dev_priv->rps.min_freq_softlimit,
 			    dev_priv->rps.max_freq_softlimit);
 
-	gen6_set_pm_mask(dev_priv, pm_iir, new_delay);
 	dev_priv->rps.last_adj = new_delay - dev_priv->rps.cur_freq;
 
 	if (IS_VALLEYVIEW(dev_priv->dev))
diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index 0495d91..ce8637d 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -4824,9 +4824,6 @@ static void valleyview_enable_rps(struct drm_device *dev)
 
 	valleyview_set_rps(dev_priv->dev, dev_priv->rps.efficient_freq);
 
-	dev_priv->rps.rp_up_masked = false;
-	dev_priv->rps.rp_down_masked = false;
-
 	/* enable WA for RC6+turbo to work together */
 	dev_priv->rps.use_RC0_residency_for_turbo = true;
 
-- 
1.7.9.5

